genstudio-create-context-addon/package.json

```diff
@@ -4,22 +4,15 @@
   "private": true,
   "dependencies": {
     "@adobe/aem-cf-editor-ui-ext-tpl": "^2.1.0",
-    "@adobe/aio-sdk": "^6.0.0",
-    "@adobe/exc-app": "^0.2.21",
```

Comment:
i remember this one being important. is it bc we use it in the SDK now?

Reply:
ah i think we use this in create.

Reply:
Deployed to dev-console stage workspace and test with local in GS Eng 01.